metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -42,85 +42,9 @@ const (
 	httpResponseCodeClientTimeout = -1
 )
 
-type StackdriverSink struct {
```

Comment:
Why move?

Reply:
By mistake, when removing and reintroducing code from below.